app/database/db.go

```diff
@@ -37,24 +37,26 @@ func Open(source interface{}) (*DB, error) {
 }
 
 func (conn *DB) inTransaction(txFunc func(*DB) error) (err error) {
+	var transactionStarted = false
 	var txDB = conn.db.Begin()
-	if err != nil {
-		return err
+	if txDB.Error != nil {
```

Comment:
Haha it was so stupid 👍  (due to a change to the initial code). Such a shame that Go does not help spotting this kind of mistake.